tests/checker_test.py

```diff
@@ -397,21 +394,16 @@ def test_code_block_without_language_is_works_with_sphinx_pre310(
 """
         ignores = types.construct_ignore_dict()
         # fmt: off
```

Comment:
All 6 remaining `fmt` comments can be removed. They were there to fixate the empty line between the now removed context manager and the variable assignement.